README.md

```diff
@@ -94,7 +97,7 @@ def character_number_is_even(response):
 validating_runner = Runner(
     openai.ChatCompletion.create,
     resources,
-    validation_function=character_number_is_even,
+    validators=character_number_is_even,
```

Comment:
Nit on naming: plural suggests that multiple (or at least a list) are required. Is there any way to name it better? (I don't really know right now)

Also, technically we're breaking compatibility, so SemVer-wise it should be a major release 😅

Reply:
I propose moving to `validation`, not implying singular or plural. Ok, so let's make it a major release then 😅